AzureMonitor: Document maximum returned records for Resource Graph (#49801)
grafana/grafana: docs/sources/datasources/azuremonitor/_index.md

  If using the `All` option, then check the `Include All Option` checkbox and in the `Custom all value` field type in the following value: `all`. If `$myVar` has value `all` then the macro will instead expand to `1 == 1`. For template variables with a lot of options, this will increase the query performance by not building a large "where..in" clause.

### Working with large Azure resource data sets

> If a request exceeds the [maximum allowed value of 1000 records](https://docs.microsoft.com/en-us/azure/governance/resource-graph/concepts/work-with-data#paging-results), the result will be paginated but only the first page of results will be returned. You can use filters to reduce the amount of records returned under that value.

Reviewer: ```suggestion
If a request exceeds the [maximum allowed value of 1000 records](https://docs.microsoft.com/en-us/azure/governance/resource-graph/concepts/work-with-data#paging-results), the result is paginated and only the first page of results are returned. You can use filters to reduce the amount of records returned under that value.
```
